Per-component release notes PRs in the release-notes skill (#10382) (#10383)
dotnet/core: .github/skills/release-notes/SKILL.md

4. `api-diff` / `dotnet-inspect` verifies public APIs and confirms suspect features still exist in the shipped build
5. `release-notes` writes curated markdown using the higher-value entries from `features.json`
6. `review-release-notes` runs a final multi-model editorial QA pass against the scoring rubric and examples
7. Output is a set of pull requests per release milestone in dotnet/core: a base PR that holds shared metadata (`changes.json`, `features.json`, `README.md`, `build-metadata.json`) and one PR per component file. Each component PR targets the base branch so component teams review and edit their file in isolation. See [`pr-layout.md`](references/pr-layout.md) for the full layout and naming scheme.

## Local testing (no PRs)

> To dry-run the skill against a milestone, have the agent create the branch set locally and skip `gh pr create` (and `git push`).

Author: ```suggestion
To dry-run the skill against a milestone, only create the branch set locally. Don't push the branches or create the PRs.
```
